src/components/AddItemForm.js

```diff
@@ -45,6 +47,13 @@ class AddItemForm extends React.Component {
       item => item.name_normalized === name_normalized,
     );
 
+    const invalid = name_normalized.length > 0;
```

Comment:
I'm sorry...one more tiny change. : ).  This would make more sense with a different variable name, I think. Right now it returns true if the name has length, which is a _valid_ name.

Maybe `const validItemName = name_normalized.length > 0;`

and then in the if check below would be `if(!validItemName)....` etc.